examples/demo-apps/apple_ios/LLaMA/LLaMA/SupportingFiles/Debug.xcconfig

```diff
@@ -16,7 +16,8 @@ OTHER_LDFLAGS = $(inherited) \
 // Include headers and libraries from $(TEMP_DIR)/cmake for it.
 HEADER_SEARCH_PATHS = $(inherited) \
     $(SRCROOT)/../../../../.. \
-    $(TEMP_DIR)/cmake/include
+    $(TEMP_DIR)/cmake/include \
+    $(SRCROOT)/../../../../extension/llm/tokenizers/include 
```

Comment:
This already looks much better!
Can we do something to the build script get `cmake install` copy those headers to `$(TEMP_DIR)/cmake/include` for us?

Reply:
Can we merge this and then add this feature? We want to unbreak trunk ASAP.

Reply:
Please go ahead with merging and let me know when the follow up is ready :)